src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php

```diff
@@ -328,6 +341,12 @@ private function getListenerInfo($listener, $eventName)
                 $listener = array($listener, '__invoke');
             }
             $class = is_object($listener[0]) ? get_class($listener[0]) : $listener[0];
+
+            $listenerHash = md5($class);
```

Comment:
Thanks for noticing! In de990fe I've changed how the hash is computed. I'm going to re-profile and I'll update the comparison if it changes.